package/gluon-respondd/files/etc/hotplug.d/iface/10-gluon-respondd

```diff
@@ -6,29 +6,17 @@
 DEVLIST=/var/run/gluon-respondd.devs
 
 ifname_to_dev () {
-	json_load "$(ubus call network.interface.$1 status)"
-	json_get_var dev device
+  json_load "$(ubus call network.interface.$1 status)"
+  json_get_var dev device
 
-	echo "$dev"
+  echo "$dev"
```

Comment:
Please keep the indentation as tabs. I know it's pretty inconsistent across Gluon, but lately I've tried to move from spaces to tabs in files I touched.